linera-base/src/crypto/mod.rs

```diff
@@ -11,14 +11,27 @@ mod secp256k1;
 use std::{io, num::ParseIntError};
 
 use alloy_primitives::FixedBytes;
-pub use ed25519::{
-    Ed25519PublicKey as PublicKey, Ed25519SecretKey as SigningKey, Ed25519Signature as Signature,
-};
 use ed25519_dalek::{self as dalek};
 pub use hash::*;
 use serde::{Deserialize, Serialize};
 use thiserror::Error;
 
+/// The public key of an authority.
+pub type AuthorityPublicKey = ed25519::Ed25519PublicKey;
+/// The private key of an authority.
+pub type AuthorityPrivateKey = ed25519::Ed25519SecretKey;
```

Comment:
Let's use the term "secret key", because "private key" can be abbreviated PK, which is confusing.

But maybe we should only use key _pairs_ almost everywhere anyway?

Reply:
Renamed.